1904-12-30.xml

```diff
@@ -0,0 +1,2634 @@
+                        <p>C. Loukaitis &amp; Co., Government Purveyors</p>
+                        <p>Garage-Boulac Street, Cairo.</p>
+                        <p>24796-28-4-905</p>
+                    </div>
+                    <div type="item">
+                        <p>TENDERS are invited for the supply of PROVISIONS to the Ellerman Line of
+                            Steamers for the year 1905. Apply to Mr. N. E. Tamvaco, Agent, Rue
+                            Sesostric, Alexandria.</p>
+                        <p>24796-28-4-905</p>
+                    </div>
+                    <cb n="2"/>
+                    <div type="advert">
+                        <head>ALDERSON'S GARDEN, BULKELEY</head>
+                        <p>Friday, December 30, at 4 p.m.</p>
+                        <p>GRAND VARIETY ENTERTAINMENT</p>
+                        <p>BY THE</p>
+                        <p>ROYAL TOOKEY FAMILY</p>
+                        <p>PROFESSOR DR. WHOISIT</p>
+                        <p>The Great Magician</p>
+                        <p>The Grand Firework</p>
+                        <p>Christmas Tree</p>
+                        <p>The Comic Sketch</p>
+                        <p>The Conscript</p>
+                        <p>Dancing after the Concert.</p>
+                        <p>God Save the King</p>
+                        <p>British Garrison with their families and officers of mercantile marine in
+                            harbour and holders of permits are invited.</p>
+                        <p>25,086-3-3</p>
+                    </div>
+                </div>
+
+                <cb n="3"/>
+                <div type="advert" xml:id="deg-ad-hcf01">
+                    <head>H &amp; C FLICK <lb/>CIGAR IMPORTERS.</head>
+                    <p>Largest depot of cigars in Egypt. Always in stock the best brands of Havana
+                        such as Henry Clay, Bock, Murias, Lopez, I. Alvarez, Cabanas, Villar y
+                        Villar, etc., etc.</p>
+                    <p>Manilla, Hamsburg, and Holland "Louis Tiuchant" Cigars.</p>
+                    <p>Smoking Tobacco. Articles for smokers.</p>
+                    <p>Finest Egyptian Cairo Cigarettes, own manufacture.</p>
+                    <p>Stores for Retail Sales:</p>
+                    <p>Cairo</p>
+                    <p>Havanna House</p>
+                    <p>Grand Continental Hotel Building.</p>
+                    <p>Alexandria</p>
+                    <p>Old Bourse Street</p>
+                    <p>St. Mark's Buildings</p>
+                    <p><measure type="indexNo">25051-12.4.905</measure></p>
+                </div>
+                <div type="advert">
+                    <head>FORCE APRES
+                        <?oxy_comment_start author="ajk22" timestamp="20221007T170708-0400" comment="Needs template."?>MALADIE<?oxy_comment_end?></head>
+                </div>
+                <div type="section">
```

Comment:
no need to use a section here--just have the items it contains stand on their own.